modules/programs/autorandr.nix

```diff
@@ -267,7 +279,8 @@ let
           + concatMapStringsSep "," toString (flatten config.transform))
         ++ optional (config.scale != null)
         ((if config.scale.method == "factor" then "scale" else "scale-from")
-          + " ${toString config.scale.x}x${toString config.scale.y}"))
+          + " ${toString config.scale.x}x${toString config.scale.y}")
+        ++ optional (config.extraConfig != null) config.extraConfig)
```

Comment:
```suggestion
        + config.extraConfig)
```

Reply:
Well I guess my new amend could maybe cause an issue with an extra \n at the end?
I thought I ran the tests right but wasn't sure. I'll double check

Reply:
OK so not using `optional` here would make the autorandr scale test fail bc it would be introducting a newline for empty `extraConfig`. So I think using `optional` with the condition being `config.extraConfig != ""` is most correct.